src/challeng.cpp

```diff
@@ -426,9 +411,24 @@ void do_duel(CHAR_DATA *ch, const char *argument) {
     send_to_char("|cYou did not challenge that person.|w\n\r", ch);
 }
 
+namespace {
+void tell_results_to(int who) {
+    if (who == 0) {
```

Comment:
I know you only moved it, but the who 0/1 thing is non-obvious.  We can redo it separately.

Reply:
Totally: an `enum class` or similar...or anything...